client/css/style.css

```diff
@@ -2,6 +2,11 @@
 @import "bootstrap.css";
 @import "../../node_modules/primer-tooltips/build/build.css";
 
+:root {
```

Comment:
put this in a separate constants file and import it maybe?

Reply:
Good idea, I'll do that as part of #2315 though.

I don't know about other editors, but vars integration in Atom is _really_ cool! I'm wondering how good it stays when putting vars in a different files (I know this isn't TL's problem at all if it doesn't).